modules/admin_manual/pages/configuration/user/oidc/oidc.adoc

```diff
@@ -48,13 +48,49 @@ Setting up ownCloud Server to work with OpenID Connect requires a couple of comp
 +
 * `'http.cookie.samesite' \=> 'None',`
 +
-See `config.sample.php` and {schemeful-samesite-url}[Schemeful Same-Site] for examples and details.
+See xref:configuration/server/config_sample_php_parameters.adoc#define-how-to-relax-same-site-cookie-settings[config.sample.php] and {schemeful-samesite-url}[Schemeful Same-Site] for examples and details.
 +
 * Settings for the OpenID Connect App
 +
-See `config.apps.sample.php` for examples and details.
+See xref:configuration/server/config_apps_sample_php_parameters.html#app-openid-connect-oidc[config.apps.sample.php] for examples and details or see section xref:settings-in-database[Settings in database] below when running clustered setups.
 - Service discovery for the xref:owncloud-desktop-and-mobile-clients[ownCloud Clients]
 
+=== Settings in Database
+
+In case you run a clustered setup, the following method is preferred because it is stateless. The OpenID Connect app checks for settings in the database first. If none is found, it falls back to the settings stored in `config.php`. If a malformed JSON string is found, an error is logged. You have to store your settings as JSON formatted string in the ownCloud database table `oc_appconfig` with following keys: 
+
+[width="50%",cols="40%,50%",options="header"]
+|===
+| Key
+| Value
+
+| appid
+| 'openidconnect'
+
+| configkey
+| 'openid-connect'
+
+| configvalue
+| _JSON-String_
+|===
+
+The _key->value_ pairs are the same as when storing them to the `config.php` file. This task has to be done by opening the database console for your ownCloud database and enter the following example command. Use the database commands `UPDATE` or `DELETE` to change or delete this keys.
```

Comment:
Do we really want people doing direct database commands?
Can they do something like:
```
occ config:app:set openidconnect openid-connect --value='{"provider-url":"https:\/\/idp.example.net","client-id":"fc9b5c78-ec73-47bf-befc-59d4fe780f6f","client-secret":"e3e5b04a-3c3c-4f4d-b16c-2a6e9fdd3cd1","loginButtonName":"Login via OpenId Connect"}'
```

Reply:
Q: is the `occ config:app:set` command writing into the database?
If yes, I would definitely go for this approach instead writing directly to the DB...

Reply:
> Q: is the occ config:app:set command writing into the database?

yes, those config set/get/delete commands manage the database.

Reply:
👍 will change the text accordingly

Reply:
SQL commands have been removed and replaced by config:app:set